modules/javafx.controls/src/main/java/javafx/scene/control/skin/SliderSkin.java

```diff
@@ -407,7 +407,7 @@ public Object queryAccessibleAttribute(AccessibleAttribute attribute, Object...
 
         thumb.setOnMouseExited(me -> {
             Tooltip t = getSkinnable().getTooltip();
-            if (t != null && t.isAutoHide()) {
+            if (t != null && t.isAutoHide() && !t.consumeAutoHidingEventsProperty().isBound()) {
```

Comment:
I still think a second property or a boolean might be better.  What if the app code binds autoHidingEventsProperty after onMouseEntered?  I know this is highly unlikely, but there is a potential.

At the same time, I  think we should proceed with this fix as it addresses a problem which occurs always.

Reply:
Agreed. Let's file a follow-on bug to consider the larger question of whether we should change the default value of `consumeAutoHidingEventsProperty` for Tooltip, so that they will not consume auto-hiding events by default.

Reply:
I have filed [JDK-8298529](https://bugs.openjdk.org/browse/JDK-8298529) for this.